sources/packages/web/src/views/aest/StudentApplicationView.vue

```diff
@@ -52,23 +63,150 @@ export default defineComponent({
   },
   setup(props) {
     const { emptyStringFiller } = useFormatters();
-    const applicationDetail = ref({} as ApplicationBaseAPIOutDTO);
+    const { searchByKey } = useFormioUtils();
+    const applicationDetail = ref({} as ApplicationSupplementalDataAPIOutDTO);
     const initialData = ref({});
     const selectedForm = ref();
+    let applicationWizard: FormIOForm;
 
+    /**
+     * Happens when all the form components are rendered, including lists.
+     */
+    const formRender = async (form: FormIOForm) => {
+      applicationWizard = form;
+      // Highlight changes in the form after all the components are rendered.
+      // List components are ready only after the form is rendered.
+      highlightChanges();
+    };
+
+    /**
+     * Loads the initial application data.
+     */
     onMounted(async () => {
       // When the application version is present load the given application version instead of the current application version.
       const applicationId = props.versionApplicationId ?? props.applicationId;
+      const application = await ApplicationService.shared.getApplicationDetail(
+        applicationId,
+      );
       applicationDetail.value =
-        await ApplicationService.shared.getApplicationDetail(applicationId);
+        application as ApplicationSupplementalDataAPIOutDTO;
       selectedForm.value = applicationDetail.value.applicationFormName;
       initialData.value = {
         ...applicationDetail.value.data,
         isReadOnly: true,
       };
+      highlightChanges();
```

Comment:
Having this method called at line 79 alone is not enough?

Reply:
Calling only from the render now.